### DISCUSSION TITLE
brain segmentation

### DISCUSSION THREAD

**Discussion by ggrrll** [Project-MONAI/MONAI#4829] [Q&A]
Hi, 

I was wondering if you have a pre-trained model for brain segmentation, for instance like 
https://github.com/Deep-MI/FastSurfer

I just found https://github.com/Project-MONAI/model-zoo/tree/dev/models/brats_mri_segmentation 

thanks

**Comment by Nic-Ma** [comment]
Hi @ggrrll ,

You can download the pretrained weights from the model:
https://github.com/Project-MONAI/model-zoo/releases/tag/hosting_storage_v1

Thanks.

**Reply by ggrrll** [reply]
thanks, but my Q was about what can be done with that : is there a ready-to-deploy model that does the full brain segmentation? (like in fastsurfer) I guess no, right? (we would have to re-train for instance [this one](https://github.com/Project-MONAI/model-zoo/tree/dev/models/brats_mri_segmentation) with a proper new dataset )

**Comment by SoheilaBehrouz** [comment]
I am going to find a pre trained model that can segment glioma tumor in brain in T2 weighted images. is there any model?

**Comment by ggrrll** [comment]
in case, I stopped working on the topic  almost 2y ago ; therefore I won´t follow up on this 

maybe 

https://github.com/AndreasGkatz

https://github.com/grodgers1

are interested
